packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -140,6 +140,55 @@
             </A.Generic>
           </Hds::Alert>
         </SF.Item>
+
+        <SF.Item>
+          <Hds::Alert @type="inline" @color="success" as |A|>
+            <A.Title>An alert with an HTML link compared to the <code>Hds::Link</code> in the description</A.Title>
+            <A.Description>
+              <p>Description with an <a href="#">HTML link</a>.</p>
+              <p>
+                Compared with a
+                <Hds::Link::Inline @href="#">Primary Hds::Link::Inline</Hds::Link::Inline>, and a
+                <Hds::Link::Standalone @icon="hashicorp" @text="Primary Hds::Link::Standalone with an icon" @href="#" />
+              </p>
+              <p>
+                Also compared with a
+                <Hds::Link::Inline @href="#" @color="secondary">Secondary Hds::Link::Inline</Hds::Link::Inline>, and a
+                <Hds::Link::Standalone
+                  @icon="hashicorp"
+                  @color="secondary"
+                  @text="Primary Hds::Link::Standalone with an icon"
+                  @href="#"
+                />
+              </p>
+            </A.Description>
+          </Hds::Alert>
+        </SF.Item>
+
+        <SF.Item>
+          <Hds::Alert @type="compact" @color="success" as |A|>
+            <A.Title>An alert with an HTML link compared to the <code>Hds::Link</code> in the description</A.Title>
+            <A.Description>
+              Compact alert with an
+              <a href="#">HTML link</a>.
+              <p>
+                Compared with a
+                <Hds::Link::Inline @href="#">Primary Hds::Link::Inline</Hds::Link::Inline>, and a
+                <Hds::Link::Standalone @icon="hashicorp" @text="Primary Hds::Link::Standalone with an icon" @href="#" />
+              </p>
+              <p>
+                Also compared with a
+                <Hds::Link::Inline @href="#" @color="secondary">Secondary Hds::Link::Inline</Hds::Link::Inline>, and a
+                <Hds::Link::Standalone
+                  @icon="hashicorp"
+                  @color="secondary"
+                  @text="Primary Hds::Link::Standalone with an icon"
+                  @href="#"
+                />
+              </p>
+            </A.Description>
+          </Hds::Alert>
+        </SF.Item>
```

Comment:
Let me know if you have better ideas for the new Showcase examples

Reply:
Thanks, I thought about doing that. I think I'll go ahead and break those 2 examples out as you suggested.

Reply:
Pushed an update